demon.js

```diff
@@ -232,6 +282,9 @@ data.forEach(function (links) {
 // Comment back in to observe impfstoff.link for availabilities.
 // observeImpfstoff();
 
+// Comment back in order to observe Punctum Medico
+// observePunctumMedico();
```

Comment:
assuming #3 merges, we should default that one on

Reply:
yes